lib/query-interface.js

```diff
@@ -45,12 +48,13 @@ module.exports = (function() {
 
     options = Utils._.extend({
       raw: true,
-      type: this.sequelize.QueryTypes.SELECT
+      type: this.sequelize.QueryTypes.SELECT,
+      logging: false
     }, options || {});
 
     var showSchemasSql = self.QueryGenerator.showSchemasQuery();
 
-    return this.sequelize.query(showSchemasSql, options).then(function(schemaNames) {
+    return this.sequelize.query(showSchemasSql, undefined, options).then(function(schemaNames) {
```

Comment:
`options` is fine as the second parameter to `query`, dont need to pass undefined to callee.